service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/ServiceClient.java

```diff
@@ -134,20 +139,16 @@ public void send(String deviceId, Message message) throws IOException, IotHubExc
      */
     public void send(String deviceId, String moduleId, Message message) throws IOException, IotHubException
     {
-        // Codes_SRS_SERVICE_SDK_JAVA_SERVICECLIENT_28_001: [The function shall throw IOException if the member AMQP sender object has not been initialized]
-        if (this.amqpMessageSender == null)
-        {
-            throw new IOException("AMQP sender is not initialized");
-        }
-        // Codes_SRS_SERVICE_SDK_JAVA_SERVICECLIENT_28_002: [The function shall call send() on the member AMQP sender object with the given parameters]
-        this.amqpMessageSender.send(deviceId, moduleId, message);
+        AmqpSend amqpMessageSender = new AmqpSend(hostName, userName, iotHubConnectionString, this.iotHubServiceClientProtocol, this.sasTokenExpiryTime);
+        amqpMessageSender.send(deviceId, moduleId, message);
```

Comment:
Is `close()` a part of `AmqpSend`?

Reply:
It is an API, but it does nothing anymore. Now, the amqpSend class does the open/send/close all under the send() API. I had to do this because the old Open() API didn't declare any thrown exceptions. It didn't declare any thrown exceptions because it actually didn't do any IO operations weirdly